### ISSUE TITLE
Set Link to No Follow

### ISSUE THREAD

**Issue by Tamatheunicorn** [WordPress/gutenberg#10137]
Is it possible to set Links to "No Follow" with Gutenberg? I had a Plugin where I could chose, but it doesn't seem to work with Gutenberg.
Is there a Solution?

**Comment by designsimply** [comment]
One way would be by editing the HTML.

1. Add the link.
1. Select More > Edit as HTML for the block that contains the link.
1. Add `rel="nofollow"` to the link code.

![screen shot 2018-09-24 at 11 52 34 am](https://user-images.githubusercontent.com/1119271/45969304-9228be00-bff0-11e8-814a-084d2d3c0cf5.png)

fwiw, I searched Google and found [an article](https://www.searchenginejournal.com/use-nofollow-links/245241/) where they recommend adding nofollow for cases like ads, sponsors, and paid links and, while I am not an SEO expert, my read on it suggests that it's probably just fine to keep links you are adding manually follow-able.

Also, if you can link to the plugin you're using, I can help test it and make suggestions for how/where to post to help make sure the plugin authors are aware of the problem.

**Comment by Tamatheunicorn** [comment]
You added a Block a paragraph, but I work with the normal version. I cannot change it to html - at least I haven't found a Button to change it html yet.
![image](https://user-images.githubusercontent.com/43542146/45977092-d0ea6280-c048-11e8-852a-48461f58e520.png)
Usually I had a Button, where I edited the Link, to chose to set it as "no Follow" or "do Follow"
I used the Plugin "Ultimate Nofollow" by 5fifty before.
